executor/services/attachment_downloader.py

```
@@ -187,9 +189,7 @@ def _download_single(self, att: Dict[str, Any]) -> Dict[str, Any]:
 
         except requests.exceptions.Timeout:
             error_msg = "Download timeout"
-            logger.error(
-                f"Timeout downloading attachment '{filename}': {error_msg}"
-            )
+            logger.error(f"Timeout downloading attachment '{filename}': {error_msg}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)